docs/README.adoc

```diff
@@ -0,0 +1,80 @@
+= Building WildFly Documentation
+:toc:
+:toclevels: 2
+:icons: font
+:source-highlighter: coderay
+
+== Overview
+
+The `docs` module is not in the module listing of the parent POM. Therefore to build documentation you must be in the
+`docs` directory.
```

Comment:
You can also be in the source root and activate the docs profile in the parent pom:

mvn install -pl docs -Ddocs=true

Reply:
I missed that profile, thanks!